components/Menu.js

```diff
@@ -1,15 +1,27 @@
 import React from 'react';
-import { View } from 'react-native';
+import { View, StyleSheet } from 'react-native';
 import { BottomSheet } from 'react-native-elements';
 import { MenuItem } from './MenuItem';
 
 export const Menu = (props) => {
   return (
-    <BottomSheet isVisible>
-      <View>
-        {props.menuItems.map((menuitem, idx) => <MenuItem key={idx} item={menuitem} />)}
-        <MenuItem item={{title: 'Cancel'}} />
+    <BottomSheet
+      isVisible={props.isVisible}
+      containerStyle={styles.backdrop}
+    >
+      <View style={styles.list}>
+        {props.menuItems.map((menuitem, idx) =>
+          <MenuItem key={idx} item={menuitem} />)}
       </View>
     </BottomSheet>
   );
 };
+
+const styles = StyleSheet.create({
+  backdrop: {
+    backgroundColor: 'rgba(77, 77, 77, 0.9)'
+  },
+  list: {
+    alignItems: 'center'
+  }
+});
```

Comment:
We are currently using seperate style sheets for components, I think we should stick with doing it one way or the other, which way do you prefer?